packages/experiments/src/components/Tile/Tile.scss

```diff
@@ -253,54 +253,61 @@ button.check {
   }
 }
 
-@mixin preserveExtenders($topMargin: 0px) {
-  padding-top: 8px;
-  margin-top: $topMargin - 8px;
-  @include margin-left(-8px);
-  @include padding-left(8px);
-}
-
 .name {
-  display: block;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
   font-weight: $ms-font-weight-regular;
   color: $ms-color-neutralPrimary;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
 
+  @include margin-left(-8px);
+  @include padding-left(8px);
+
   &,
   .isLarge & {
     font-size: $ms-font-size-m;
-    height: 17px;
-    line-height: 17px;
+    height: 30px;
+    margin-top: -7px;
```

Comment:
These adjustments seem a bit random, can you specify what negative margins are doing?

Reply:
Add comments...as someone coming in fresh to this css I would be very leery of this and quick to redo it. But there are probably good reasons here...